### PR TITLE
Add ansible code to install components on linux jumpboxes

### PR DESCRIPTION
**Pull request by rukawata**
Add roles to install HANA studio and HANA client on all Linux jumpboxes except RTI.

### CODE REVIEW

File: deploy/v2/ansible/group_vars/all.yml [hunk 1]
```diff
@@ -3,12 +3,14 @@ disk_dict: {}
 start: 0
 azure_files_mount_path: "/sapmnt"
 components_list: []
+linux_comp_list: []
```
Comment (pabowers) [inline deploy/v2/ansible/group_vars/all.yml]: what's the difference between `linux_comp_list` and `components_list`?

File: deploy/v2/ansible/roles/components-install/tasks/hana_client_linux.yml [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 
-# Install HANA Client 
 - name: Create dictionary with installation variant for HANA Client 
   set_fact:
     client: "{{ item }}"
   loop: "{{ components }}"
   when: item.component == 'hana_client'
 
-- name: Install Hana Client
+- name: Install Hana Client on Linux
```
Comment (pabowers) [inline deploy/v2/ansible/roles/components-install/tasks/hana_client_linux.yml]: I assume this is the linux jump host. Can the name please reflect that?

File: deploy/v2/ansible/roles/linux-jumpboxes-install/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+
+# Collect the component list from all linux jumpboxes except rti
```
Comment (pabowers) [inline deploy/v2/ansible/roles/linux-jumpboxes-install/tasks/main.yml]: what is rti?

File: deploy/v2/ansible/group_vars/all.yml (hunk 1 above)
Comment (rukawata) [reply deploy/v2/ansible/group_vars/all.yml]: > what's the difference between `linux_comp_list` and `components_list`?

`components_list` is the list of components to be installed on HANA servers.
`linux_comp_list` is the list of components to be installed on Linux jumpboxes.

We could change components_list to hdb_comp_list, but I would prefer to do that in a separate PR.

File: deploy/v2/ansible/roles/linux-jumpboxes-install/tasks/main.yml (hunk 3 above)
Comment (rukawata) [reply deploy/v2/ansible/roles/linux-jumpboxes-install/tasks/main.yml]: > what is rti?

rti (runtime instance) is basically a bastion host that we use to run ansible-playbook.
We plan to have it destroyed (Eg. with terraform destroy) after the deployment finishes.

File: deploy/v2/ansible/roles/components-install/tasks/hana_client_linux.yml (hunk 2 above)
Comment (rukawata) [reply deploy/v2/ansible/roles/components-install/tasks/hana_client_linux.yml]: > I assume this is the linux jump host. Can the name please reflect that?

Sure. Done :)

File: deploy/v2/ansible/group_vars/all.yml (hunk 1 above)
Comment (pabowers) [reply deploy/v2/ansible/group_vars/all.yml]: sure, as long as it will be more clear in the future

**Pull request merged**